pcap.c

```diff
@@ -551,8 +551,8 @@ pcap_list_tstamp_types(pcap_t *p, int **tstamp_typesp)
 		**tstamp_typesp = PCAP_TSTAMP_HOST;
 		return (1);
 	} else {
-		*tstamp_typesp = (int*)calloc(sizeof(**tstamp_typesp),
-		    p->tstamp_type_count);
+		*tstamp_typesp = (int*)malloc(p->tstamp_type_count *
+			sizeof(**tstamp_typesp));
```

Comment:
This is the exact reverse of the point of this commit. Removing this initialization has potential to introduce a bug.

Reply:
I have no idea how that got there.

Reply:
memset takes care of that